lib/tape/notifiers/slack.rb

```diff
@@ -0,0 +1,78 @@
+require 'slack-notifier'
+
+class SlackNotifier
+  def initialize(webhook_url, deploy_info)
+    @notifier = Slack::Notifier.new webhook_url
+    @notifier.username = 'Tape'
+    @deploy_info = deploy_info
+  end
+
+  def update(status)
+    @status = status
+    @notifier.ping
+      "",
```

Comment:
I'm guessing this is because there's a comment on the next line. I think it's okay how it is.